src/Sulu/Bundle/AdminBundle/Resources/js/components/BlockCollection/SortableBlocks.js

```diff
@@ -2,11 +2,13 @@
 import React from 'react';
```

Comment:
i noticed that this file is named `SortableBlocks.js` and the respective css file is named `sortableBlockList.scss`. should we rename this file to `SortableBlockList.js`?

Reply:
I would say that makes sense, yeah :slightly_smiling_face: